docs/content/integrations/databricks.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: "Using Dagster with Databricks"
+description: Dagster can orchestrate Databricks jobs alongside other technologies.
+---
+
+# Using Databricks with Dagster
+
+Dagster can orchestrate your Databricks jobs, making it easy to chain together multiple Databricks jobs, as well as orchestrate Databricks alongside your other technologies.
+
+---
+
+## Prerequisites
+
+To get started, you will need to install the `dagster` and `dagster-databricks` Python packages:
+
+```bash
+pip install dagster dagster-databricks
+```
+
+You'll also want to have a Databricks workspace with an existing project that is deployed with a Databricks job. If you don't have one already, you can [follow the Databricks quickstart](https://docs.databricks.com/workflows/jobs/jobs-quickstart.html) to set one up.
+
+To manage your Databricks job from Dagster, you'll need three values:
+
+1. A `host` for connecting with your Databricks workspace, stored in an environment variable `DATABRICKS_HOST`, and
+2. A `token` corresponding to a personal access token for your Databricks workspace, stored in an environment variable `DATABRICKS_TOKEN`.
+3. A `DATABRICKS_JOB_ID` for the Databricks job you want to run.
+
+You can follow the [Databricks API authentication instructions](https://docs.databricks.com/dev-tools/python-api.html#step-1-set-up-authentication) to retrieve these values.
+
+---
+
+## Step 1: Connecting to Databricks
+
+The first step in using Databricks with Dagster is to tell Dagster how to connect to your Databricks workspace using a Databricks [resource](/concepts/resources). This resource contains information on where your Databricks workspace is located and any credentials sourced from environment variables that are needed to access it.
```

Comment:
Should we add a link to the `dagster-databricks` API docs in here? Maybe to `/_apidocs/libraries/dagster-databricks#dagster_databricks.DatabricksClient`?